src/shell/shell_cmds.cpp

```diff
@@ -810,11 +810,9 @@ void DOS_Shell::CMD_DIR(char* args)
 {
 	HELP("DIR");
 
-	auto envvar = GetEnvStr("DIRCMD");
-	if (envvar){
-		auto line = *envvar;
-		std::string::size_type idx = line.find('=');
-		std::string value=line.substr(idx +1 , std::string::npos);
+	auto line = std::string{};
```

Comment:
Yeah, or `std::string line = ""` looks less funky and is a less roundabout way ("I ask for `auto`... then change my mind and specify the type anyway 😎")

Reply:
Yeah ok, it's a minor thing, my biggest problem was the braces 😄